controls/datepicker/dateonly.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Date Only Support
```

Comment:
In this case, I suggest keeping the original naming as "Date Only" doesn't appear to be a common term and it makes sense only when used as a name for the struct.
```suggestion
title: Using DateOnly
```